client/notes.md

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+const compress = require('compression');
+const controller = require('./controller.js');
+
+const port = 3000;
+const app = express();
+app.use(cors());
+app.use(compress());
+app.use(bodyParser.json());
+
+app.use('/scripts', express.static(path.resolve(__dirname, '../node_modules')));
+app.use('/song/:songId', express.static(path.resolve(__dirname, '../dist')));
```

Comment:
Gets a little bit difficult from here on to read with all the comments added...perhaps move the comments down or cut out what's unnecessary?